packages/react/src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -9692,8 +9692,9 @@ Map {
         "hours": "hours",
         "increment": "Increment",
         "intervalLabels": Array [],
+        "invalidDateText": "Date is required",
         "invalidNumberLabel": "Number is not valid",
-        "invalidText": "The date time entered is invalid",
```

Comment:
This could technically be a breaking change for someone's test if they are looking for this default but is probably ok.